src/app/launcher/launcher.component.ts

```diff
@@ -78,6 +83,7 @@ export class LauncherComponent implements AfterViewInit, OnInit {
     if (this.targetEnvironment !== undefined && this.targetEnvironment.length > 0) {
       this._summary.targetEnvironment = this.targetEnvironment;
     }
+    this._summary.dependencyCheck.spacePath = '/' + this.spaceName;
```

Comment:
We don't want to define a path here. In fact, this is not a valid path for OSIO -- the user name would be prepended.

Reply:
I just checked the other day, if I pass the spacePath as I have done above, all the services like github and pipelines are correctly linked to the space and they are shown on the space dashboard for the given space name.
I am not sure, If we want to prepend the user name here ?

Reply:
I'm speaking only about the text shown in the summary. Does the summary show a full path or just a space name?

If we're showing the space path in the summary, I expect the path to look more like this: /dlabrecq@redhat.com/test, where test is the space name.

If I copy /test to the browser window, that won't route to a space without also including the username.

Reply:
okay I get it now. i will make sure that we show the full path name for the space in the summary